layouts/_default/single.html

```diff
@@ -1,14 +1,31 @@
 {{ define "hero-body" }}
 <div class="container">
-  <h1 class="title">
-    {{ .Title }}
-  </h1>
-  <h2 class="subtitle">
-    {{ .Description }}
-  </h2>
-  <h3 class="subsubtitle">
-    {{ with .Params.Author }}Maintained by: {{ delimit . ", " }}{{ end }}
-  </h3>
+  <div class="columns">
```

Comment:
You are dividing the vertical space in two equally sized columns. That produces the following result:

![image](https://user-images.githubusercontent.com/38422348/105554707-3083d600-5d08-11eb-837c-cc0eb1837e25.png)

I think we want the second column (L15) to be narrow: https://bulma.io/documentation/columns/sizes/#narrow-column